ceph-build-arm64/config/definitions/ceph-build.yml

```diff
@@ -0,0 +1,62 @@
+- job:
+    name: ceph-build-arm64
+    project-type: matrix
+    disabled: true
+    defaults: global
+    display-name: 'ceph-build-arm64'
+    block-downstream: false
+    block-upstream: false
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph
+    execution-strategy:
+      combination-filter: (DIST == "xenial" && ARCH == "arm64")
+    axes:
+      - axis:
+          type: label-expression
+          name: ARCH
+          values:
+            - arm64
+      - axis:
+          type: label-expression
+          name: DIST
+          values:
+            - xenial
+
+    builders:
+      - shell: |
+          echo "Cleaning up top-level workarea (shared among workspaces)"
+          rm -rf dist
+          rm -rf venv
+          rm -rf release
+      - copyartifact:
```

Comment:
I understand this is mostly copied from `ceph-build` but we don't need to copy artifacts any more (woo hoo!)